### DISCUSSION TITLE
Osu!Lazer not launching

### DISCUSSION THREAD

**Discussion by S4rtox** [ppy/osu#15892] [Q&A] [answered]
Updated Osu!Lazer to the last realease today, after downloading the update and restarting the game found out that it no longer opens, on the task manager only pops out a couple seconds before closing itself again. Tried updating windows & drivers but no difference. Even tried reinstalling but still same problem.
[runtime.log](https://github.com/ppy/osu/files/7637217/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7637218/updater.log)
[database.log](https://github.com/ppy/osu/files/7637219/database.log)
[network.log](https://github.com/ppy/osu/files/7637220/network.log)
[performance.log](https://github.com/ppy/osu/files/7637221/performance.log)

**Comment by bdach** [comment, marked as answer]
Will be fixed next release by #15846

**Reply by S4rtox** [reply]
Are there any temporal fixes?, or do I just need to wait for next version.

**Reply by S4rtox** [reply]
Nmv, found a workaround by literally installing every update manually . -. . Thanks anyways
